src/metabase/lib/fe_util.cljc

```diff
@@ -13,7 +13,9 @@
    [metabase.lib.temporal-bucket :as lib.temporal-bucket]
    [metabase.lib.util :as lib.util]
    [metabase.mbql.util :as mbql.u]
+   [metabase.shared.formatting.internal.date-builder :as shared.formatting.internal.builder]
    [metabase.shared.util.i18n :as i18n]
+   [metabase.shared.util.internal.time-common :as shared.ut.common]
```

Comment:
Don't import this - it's internal for a reason, as I said above.

But also the only function used from here (`to-range`) is re-exported from `metabase.shared.util.time`. (See the `(shared.ns/import-fns ...)` macro at the top of that namespace.) So you can call the same function as `shared.ut/to-range`.